src/neptune/types/series/float_series.py

```diff
@@ -96,3 +100,15 @@ def unit(self):
 
     def __str__(self):
         return "FloatSeries({})".format(str(self.values))
+
+    def clear_of_unsupported_values(self):
+        cleared_values = []
+        for value in self._values:
+            if math.isinf(value) or math.isnan(value):
```

Comment:
How is it different from `if value in UNSUPPORTED_FLOATS`?

Reply:
Wanted to never miss a case, added a function for that